python/lsst/daf/butler/cli/cmd/commands.py

```diff
@@ -202,3 +202,21 @@ def query_datasets(**kwargs):
         print("")
         table.pprint_all()
     print("")
+
+
+@click.command()
+@repo_argument(required=True)
+@click.argument('input-collection')
+@click.argument('output-collection')
+@click.argument('dataset-type-name')
+@click.option("--begin-date", "-b", type=str, default=None,
```

Comment:
@n8pease might want to comment but can we not use the `-b` and `-e` short options to begin with until we get a better feel as to whether they are confusing. We use `-b` to mean butler repo in `pipetask` and we are trying not to reuse short options if we don't have to.

Reply:
I've removed them, and updated `ci_cpp_gen3` to use the full `--begin-date` style formats.